android/app/src/main/AndroidManifest.xml

```diff
@@ -15,25 +15,12 @@
       android:theme="@style/AppTheme"
       android:networkSecurityConfig="@xml/network_security_config">
 
-      <service android:name="io.invertase.firebase.messaging.RNFirebaseMessagingService">
-        <intent-filter>
-          <action android:name="com.google.firebase.MESSAGING_EVENT" />
-        </intent-filter>
-      </service>
-
-      <service android:name="io.invertase.firebase.messaging.RNFirebaseMessagingService">
-        <intent-filter>
-          <action android:name="com.google.firebase.MESSAGING_EVENT" />
-        </intent-filter>
-      </service>
-
-      <service android:name="io.invertase.firebase.messaging.RNFirebaseBackgroundMessagingService" />
-      <service android:name="io.invertase.firebase.messaging.RNFirebaseBackgroundMessagingService" />
```

Comment:
Miks nää on poistettu?

Reply:
samaa hommaa yllä olevan kanssa.
npm kutos versiossa nää on siirtyneet sinne npmmien sisälle.
https://rnfirebase.io/migrating-to-v6#removing-native-dependencies